topik/intermediaries/raw_data.py

```diff
@@ -177,6 +176,7 @@ def import_from_iterable(self, iterable, id_field="text", batch_size=500):
             list of strings, a dictionary with one key, "text" is created and
             used.
         """
+        from elasticsearch import helpers
         batch = []
         for item in iterable:
             if isinstance(item, basestring):
```

Comment:
document Corpus saving and loading (this particular line not relevant to comment)